addons/stock/views/product_views.xml

```diff
@@ -212,7 +207,8 @@
                         <button class="oe_stat_button"
                                name="action_open_quants"
                                icon="fa-building-o"
-                               type="object" attrs="{'invisible':[('type', '!=', 'product')]}">
+                               type="object" attrs="{'invisible':[('type', '!=', 'product')]}"
+                               context="{'quantity_available_locations_domain': ('internal', 'transit')}">>
```

Comment:
```XML
<button class="oe_stat_button"
             name="action_open_quants"
             icon="fa-building-o"
             type="object" attrs="{'invisible':[('type', '!=', 'product')]}"
             context="{'quantity_available_locations_domain': ('internal', 'transit')}">
```
*(I put the code here because it moved after force-push.)*

I had to do a change about the domain on location, but I had conflict with the branch of WHE. When I resolved the conflict, I preferred to reuse what WHE did.